recipes/cuquantum/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set version = "0.1.0" %}
+{% set patch_version = "30" %}
+
+package:
+  name: cuquantum
+  version: {{ version }}.{{ patch_version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuquantum/{{ version }}/local_installers/cuquantum-linux-x86_64-{{ version }}.{{ patch_version }}-archive.tar.xz   # [linux64]
+  url: https://developer.download.nvidia.com/compute/cuquantum/{{ version }}/local_installers/cuquantum-linux-sbsa-{{ version }}.{{ patch_version }}-archive.tar.xz     # [aarch64]
+  url: https://developer.download.nvidia.com/compute/cuquantum/{{ version }}/local_installers/cuquantum-linux-ppc64le-{{ version }}.{{ patch_version }}-archive.tar.xz  # [ppc64le]
+
+  sha256: 8ad8e98f14275ffe0de02574be5c86224af1c657c41baf02c16440301ffe0aae  # [linux64]
+  sha256: 3dd04cf08f1323318e0e63a7e28bc904c426ced3367aca809fba7e7beef94063  # [aarch64]
+  sha256: 0eb84eef619a1cfab5870fb585200a9869a197866685252e4ca8187322809554  # [ppc64le]
+
+build:
+  number: 2
+  skip: True  # [win or ppc64le or cuda_compiler_version != "11.2"]
+  script_env:
+    # for some reason /usr/local/cuda is not added to $PATH in ppc64le's docker image
+    - CUDA_HOME  # [ppc64le or aarch64]
+  script:
+    - mkdir -p $PREFIX/include                                            # [linux]
+    - cp -r include/* $PREFIX/include/                                    # [linux]
+    - mkdir -p $PREFIX/lib                                                # [linux]
+    - mv lib/*.so* $PREFIX/lib/                                           # [linux]
+
+    # Patch for conda-forge:
+    # According to the CUDA support matrix, we require glibc 2.27+ on aarch64-sbsa,
+    # while conda-forge is still on 2.17. However, given that powf is the only
+    # offending symbol that needs a newer glibc, it is the easist if we just patch
+    # it.
+    - patchelf --clear-symbol-version powf $PREFIX/lib/libcustatevec.so   # [aarch64]
+  ignore_run_exports_from:
+    - {{ compiler('c') }}
+    - {{ compiler('cuda') }}
+  run_exports:
+    - {{ pin_subpackage('cuquantum', max_pin='x.x') }}
+  missing_dso_whitelist:
+    - '*/libgomp.*'        # [linux]
+    - '*/libgcc_s.so*'     # [linux]
+    - '*/libcublas.so*'    # [linux]
+    - '*/libcublasLt.so*'  # [linux]
+    - '*/libcutensor.so*'  # [linux]
```

Comment:
All of these should be removed and proper dependencies added to `run`.

Reply:
@isuruf I tried locally. cublas/cublasLt come with `cudatoolkit` and cutensor with `cutensor`, and you can see they are already added to the run section, but for some reason we still got warnings for these.

Is there a specific package for OpenMP runtime?

Reply:
> but for some reason we still got warnings for these.

Yes, but the warning tells you what to do. i.e. use patchelf to add an rpath.

> Is there a specific package for OpenMP runtime?

See our docs

Reply:
lol 
```
ERROR (cuquantum,lib/libcutensornet.so): $RPATH/libcublasLt.so.11 not found in packages, sysroot(s) nor the missing_dso_whitelist.
```
I would not think it's telling me what to do... but thanks for pointers Isuru

Reply:
@isuruf How about `libgcc_s.so`? Is it from `libgcc`?

Reply:
Turns out it's not enough to just list them in `run`; they have to be added to `host` too so that when the rpath is set to `$PREFIX/lib` conda-build can find them properly.

Reply:
Yep that makes sense. So do we not need to use `patchelf` after that change?

Reply:
Good point, John! I just removed the `patchelf` line.